helm-chart/splunk-operator/crds/enterprise.splunk.com_clustermasters.yaml

```diff
@@ -1,9 +1,9 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.2
-  creationTimestamp: null
```

Comment:
this also, is it expected?

Reply:
yes default value is null, so i have removed it